.github/workflows/ci.yml

```diff
@@ -0,0 +1,20 @@
+name: Run CI checks
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
will change this to `develop`. Will leave updates to `master` for external releases.